Exception on /runchart [POST]
Traceback (most recent call last):
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seon/Desktop/CambodiaNAS/app.py", line 280, in run_python_code
    return render_template('24h_chart.html', result=result)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/flask/templating.py", line 150, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/flask/templating.py", line 131, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/jinja2/environment.py", line 939, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/Users/seon/Desktop/CambodiaNAS/templates/24h_chart.html", line 1, in top-level template code
    {% extends 'base.html' %}
  File "/Users/seon/Desktop/CambodiaNAS/templates/base.html", line 31, in top-level template code
    {% block content %}
  File "/Users/seon/Desktop/CambodiaNAS/templates/24h_chart.html", line 6, in block 'content'
    {% include 'radarchart.html' %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/flask/templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/flask/templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: radarchart.html
Exception on /runchart [POST]
Traceback (most recent call last):
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seon/Desktop/CambodiaNAS/app.py", line 280, in run_python_code
    return render_template('24h_chart.html', result=result)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/flask/templating.py", line 150, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/flask/templating.py", line 131, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/jinja2/environment.py", line 939, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/Users/seon/Desktop/CambodiaNAS/templates/24h_chart.html", line 1, in top-level template code
    {% extends 'base.html' %}
  File "/Users/seon/Desktop/CambodiaNAS/templates/base.html", line 31, in top-level template code
    {% block content %}
  File "/Users/seon/Desktop/CambodiaNAS/templates/24h_chart.html", line 6, in block 'content'
    {% include 'radarchart.html' %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/flask/templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seon/Desktop/CambodiaNAS/venv/lib/python3.12/site-packages/flask/templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: radarchart.html
